src/data/predictions.csv

```diff
@@ -0,0 +1,44 @@
+"text", "translation"
+"<color=#FF4400>【バ場・距離両方の適性が合っていない】</color>ので\n実力を100%出せないかもしれません。","<color=#FF4400>[Aptitude for both surface and distance lacking]</color>\nShe may not be able to give 100% of her ability."
+"<color=#FF4400>【バ場適性が合っていない】</color>ので\n十分に実力を発揮できないかもしれません。","<color=#FF4400>[Aptitude for surface lacking]</color> She may struggle to display her ability."
+"<color=#FF4400>【距離適性が合っていない】</color>ので\n十分に実力を発揮できないかもしれません。","<color=#FF4400>[Aptitude for distance lacking]</color> She may struggle to display her ability."
```

Comment:
![image](https://user-images.githubusercontent.com/25855364/131911830-cdbcaff0-5d05-4b78-81bd-1360bcea61d3.png)
This is referring to turf/dirt race track. Current translation sounds very awkward

Reply:
I think it's fine to replace all `surface` word with `track`?

Reply:
Is surface not the term used to refer to the track style? I remember looking for this one specifically in the other files ~~but not very hard~~.
Changing the order would certainly help.